plugins/repository.go

```diff
@@ -0,0 +1,98 @@
+package plugins
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+)
+
+const argRepository = "path"
+
+type RepositoryPlugin struct {
+	Plugin
+	Path string
+}
+
+func (p *RepositoryPlugin) IsEnabled() bool {
+	return p.Enabled
+}
+
+func (p *RepositoryPlugin) DefineCommandLineArgs(cmd *cobra.Command) error {
+	flags := cmd.Flags()
+	flags.StringP(argRepository, "", "", "scan repository folder")
```

Comment:
I don't understand why we are using `StringP` with an empty string for short name, when we can use `String` without short string.


```suggestion
	flags.String(argRepository, "", "scan repository folder")
```